test_lib/compaction.py

```diff
@@ -23,6 +38,40 @@ def from_str(cls, output_str):
             raise ValueError(err_msg) from attr_err
 
 
+def get_gc_mode(node, keyspace, table) -> None | GcMode:
+    """Get a given table GC mode
+
+    :Arguments:
+        node {str} -- ip of db_node
```

Comment:
Node is used, not just ip - documentation is wrong

Reply:
fixed

Reply:
nor here

Reply:
refresh and you will see